themes/cantera/assets/js/copybutton.js

```diff
@@ -0,0 +1,60 @@
+$(document).ready(function () {
```

Comment:
If this is copy-pasted from the Python docs, we probably need to include the license somewhere in it.

Reply:
https://github.com/python/python-docs-theme/blob/master/python_docs_theme/static/copybutton.js